src/utils/load-proto.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Load Proto
+ * @module src/utils/load-proto
+ */
+
+const grpc = require('grpc')
+const grpcProtoLoader = require('@grpc/proto-loader')
```

Comment:
Does this upgrade have to go with this PR? It seems pretty unrelated and the PR is pretty large as it is.

Reply:
@treygriffith the upgrade was already merged to `lnd-engine`. I just moved the loading of the proto file out into its own method

Reply:
I see - deleted file made the diff hard to parse.

Reply:
Yup, that was my bad. Deleted the `generate-lnd-client` file and created 2 more